android-json-form-wizard/src/main/java/com/vijay/jsonwizard/utils/FormUtils.java

```diff
@@ -646,10 +651,6 @@ public static void setEditMode(JSONObject jsonObject, View editableView, ImageVi
             boolean readyOnly = jsonObject.getBoolean(JsonFormConstants.READ_ONLY);
```

Comment:
If this key isn't found, what will happen? Can you use optBoolean instead? Then pass default as false

Reply:
Okay